TypeCobol.Test/SemanticDomainTest.cs

```diff
@@ -0,0 +1,1668 @@
+﻿using System;
+using System.Linq;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using TypeCobol.Test.Compiler.Parser;
+using System.Diagnostics;
+using System.Collections.Generic;
+using System.IO;
+using TypeCobol.Codegen.Config;
+using TypeCobol.Compiler.Diagnostics;
+using TypeCobol.Compiler.Directives;
+using TypeCobol.Test.Utils;
+using System.Text;
+using TypeCobol.Compiler.Parser;
+using TypeCobol.Tools.Options_Config;
+using TypeCobol.Compiler.Domain;
+using TypeCobol.Compiler;
+using TypeCobol.Compiler.CodeElements;
+using TypeCobol.Compiler.CodeModel;
+using TypeCobol.Compiler.Nodes;
+using TypeCobol.Compiler.Symbols;
+using TypeCobol.Compiler.Types;
+using Type = TypeCobol.Compiler.Types.Type;
+
+namespace TypeCobol.Test.Domain
+{
+    [TestClass]
+    public class SemanticDomainTest
+    {
+        public static TypeCobolConfiguration DefaultConfig = null;
+        public static ProgramSymbolTableBuilder Builder = null;
+        public static NodeListenerFactory BuilderNodeListenerFactory = null;
+        public static string DefaultIntrinsicPath = @"C:\TypeCobol\Sources\##Latest_Release##\Intrinsic\Intrinsic.txt";
```

Comment:
Yes  this is only a reference, the Intrinsic.txt file is not on Github. This is here because I started testing using this reference. You can see that AppVeyor tests don't use it. I am going  to remove this reference.

Reply:
You are right, this reference is when I started testing, but as you can see AppVeyor tests don't use this file which is not on GitHub. I am going to remove this reference, which is no longer needed.